modifiers.py

```diff
@@ -160,6 +160,22 @@ def change_function_call_name(self) -> Self:
 
         return self
 
+    def insert_empty_statements(self) -> Self:
+        """A code modifier that causes an IndentationError.
+
+        This will randomly insert empty if statements.
+
+        Returns:
+            The modifier instance.
+        """
+        total_length = len(self.file_contents)
+        random_position = random.randrange(total_length)
+
+        statement = "if kappa_cj9:\n"
```

Comment:
Taking more suggestions for this as it doesn't always have to be the same string inserted.

Reply:
We could try to generate statements randomly, but for now, I would probably just make a list and pick a random one.

Reply:
Have added a couple more in [#255e4df ](https://github.com/Vthechamp22/kindly-kappa/pull/7/commits/255e4dfaf5d62a253b5eef4e81246362a5dbbb11) but will add others as they get suggested.